backend/user-service/src/controllers/authController.ts

```diff
@@ -231,6 +234,97 @@ export async function updateBothTokens(req: Request, res: Response) {
   }
 }
 
+export const sendResetEmail: RequestHandler[] = [
+  body("email").notEmpty().isEmail(),
+  async (req, res) => {
+    if (!validationResult(req).isEmpty()) {
+      res.status(400).json({ errors: validationResult(req).array() });
+      return;
+    }
+
+    const { email } = matchedData(req);
+
+    try {
+      const user = await prisma.user.findUnique({ where: { email } });
+
+      if (!user) {
+        res.status(404).json({ errors: [{ msg: "User not found." }] });
+        return;
+      }
+
+      const resetToken = generateTemporaryToken(email);
+      const mailOptions = {
+        from: "your_email@example.com",
+        to: email,
+        subject: "Password Reset",
+        text: `Click the link below to reset your password: http://localhost:8000/reset-password?token=${resetToken}`,
```

Comment:
can we store http://localhost:8000 in an env variable?

Reply:
actually wait this would be changed to a frontend url so maybe its also ok

Reply:
yea haha i think it would be a link to the form, then the form would send the post request to this, thanks!